components/brave_wallet/common/hash_utils.h

```diff
@@ -17,9 +17,11 @@ namespace brave_wallet {
 
 inline constexpr size_t kKeccakHashLength = 32;
 inline constexpr size_t kRipemd160HashLength = 20;
+inline constexpr size_t kSHA512HashLength = 64;
```

Comment:
https://commondatastorage.googleapis.com/chromium-boringssl-docs/sha.h.html#SHA512_DIGEST_LENGTH

`sha.h` has this constant `SHA512_DIGEST_LENGTH` defined as `64`. 
We shouldn't hardcoded these ourselves. 
Likewise `RIPEMD160_DIGEST_LENGTH`.

Reply:
I didn't want to leak OpenSSL headers through this header.

Chrome is OK to hardcode this 
https://source.chromium.org/chromium/chromium/src/+/main:crypto/sha2.h;l=26?q=kSHA256Length&ss=chromium
https://source.chromium.org/chromium/chromium/src/+/main:crypto/hash.h;l=15-17?q=kSha512Size&ss=chromium